chalice/compat.py

```diff
@@ -1,13 +1,27 @@
 import os
 import platform
+import socket
 import six
 
 
 if six.PY3:
     from urllib.parse import urlparse, parse_qs
+
+    def is_broken_pipe_error(error):
+        # type: (Exception) -> bool
+        return isinstance(error, BrokenPipeError)  # noqa
```

Comment:
Pylint on python2 yelled at me for having an unknown variable name of BrokenPipeError as it only exists in Python3. Is there a better way to handle this? It seemed that using `#noqa` is the precedent that has been set in previous places for pylint errors due to need for type hinting.